_data/tracers_instrumentation.yml

```diff
@@ -68,3 +68,13 @@
   transports: "http, log file"
```

Comment:
@codefromthecrypt I run into Rust one recently [1], do you think we could add it? (apologies if it is there already but I haven't find it)

[1] https://github.com/palantir/rust-zipkin

Reply:
sounds good. Usually we don't add things unless we maintain them, or one of the authors says ok. @sfackler sound good?

Reply:
Sounds good to me!

Reply:
@reta once we merge this, make a separate PR so that watchers can notice!